ingester/database.py

```diff
@@ -339,12 +338,50 @@ def connect_datasheet_mc(self, datasheet_id, mc_id):
                 query = """
                                    MATCH (mc:ModelCard {external_id: $mc_id})
                                    CREATE (d:Datasheet {
-                                       external_id: $data_id
+                                       external_id: $data_id,
+                                       name: 'Default Datasheet'
                                    })
                                    CREATE (mc)-[:TRAINED_ON]->(d)
                                """
                 session.run(query, data_id=datasheet_id, mc_id=mc_id)
 
+    def connect_foundational_model(self, retrain_mc_id, foundational_mc_id):
+        """
+        Connects the foundational model.
+        :param retrain_mc_id:
+        :param foundational_mc_id:
+        :return:
+        """
+        with self.driver.session() as session:
+
+            model_card = self.check_model_card_exists(foundational_mc_id)
+            if model_card:
+                query = """
+                               MATCH (retrain_mc:ModelCard {external_id: $retrain_mc_id}) , (foundational_mc:ModelCard {external_id: $foundational_mc_id})
+                               CREATE (retrain_mc)-[:TRANSFORMATIVE_USE_OF]->(foundational_mc)
+                               """
+                session.run(query, retrain_mc_id = retrain_mc_id, foundational_mc_id = foundational_mc_id)
+
+    def check_model_card_exists(self, mc_id):
+        """
+        check_model_card_exists
```

Comment:
function name and the comment are the same. At the least, remove the underscores.